src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery[] lottery = new Lottery[NUMBER_OF_BALLS];
+        for (int i = 0; i < lottery.length; i++) {
+            lottery[i] = new Lottery().getRandomBall();
+            printResult(lottery[i]);
+        }
+    }
+
+    public static void printResult(Lottery lottery) {
+        System.out.println(lottery.toString());
```

Comment:
not a mistake but am not sure if there's any point in extracting such a trivial piece of logic

Reply:
I couldn't find another way in my head how to print it out)